java-checks/src/test/files/checks/StaticFieldUpateCheck.java

```diff
@@ -71,6 +71,22 @@ public int compareTo(Object o) {
     };
   }
 
+  private synchronized Comparable<Object> foo() {
+    return new Comparable<Object>() {
+      @Override
+      public int compareTo(Object o) {
+        value = 0; // FN: "value" may not be assigned inside a static method
```

Comment:
To be verified that it was NOT covered before. Otherwise it's a regression.

Reply:
I verified: this one (and the next one added in this file) were already FN before the change.